lib/zhr_devs/identity_management/identity.ex

```diff
@@ -26,6 +26,19 @@ defmodule ZhrDevs.Identity do
     GenServer.start_link(__MODULE__, success_struct, name: via_tuple(hashed_identity))
   end
 
+  @spec renew_login(binary()) :: :ok | no_return()
+  def renew_login(hashed_identity) when is_binary(hashed_identity) do
+    GenServer.call(via_tuple(hashed_identity), :renew_login)
+  end
+
+  @spec digest_to_urlsafe!(binary()) :: Uptight.Result.t()
+  def digest_to_urlsafe!(digest) when is_binary(digest) do
+    hashed_identity = Uptight.Base.mk_url(digest)
+    assert Result.is_ok?(hashed_identity), "Hashed identity can't be converted to urlsafe"
```

Comment:
I don't think this check is even needed, you're crashing when it's not `is_binary` anyway.

Reply:
In other words: it's not your concern that some bug may exist in `Uptight.Base` which shall prevent you from converting a binary to urlsafe, right? So just drop assert and return `digest |> Base.mk_url() |> Result.from_ok()`.